### PR TITLE
Fix tests for OpenSUSE 15.1

### PR DESCRIPTION
**Pull request by relrod**
##### SUMMARY

Fallout from https://github.com/ansible/ansible/issues/68783

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

- setup_mysql_db tests
- zypper tests

### CODE REVIEW

**Comment by felixfontein** [comment]
@relrod thanks! Can you also re-enable the zypper_repository tests, i.e. revert https://github.com/ansible-collections/community.general/pull/96 ? Thanks :)

File: tests/integration/targets/setup_mysql_db/tasks/main.yml [hunk 1]
```diff
@@ -63,6 +63,13 @@
   when: ansible_facts.pkg_mgr == 'apt'
   notify: cleanup mysql
 
+- name: install mysqldb_test opensuse dependencies
+  zypper:
+    name: "{{ mysql_packages }}"
+    state: latest
+  when: ansible_facts.pkg_mgr == 'zypper'
```
Comment (felixfontein) [inline tests/integration/targets/setup_mysql_db/tasks/main.yml:70]: For now (until ansible/ansible#67684 is merged), this needs to be:
```suggestion
  when: ansible_facts.pkg_mgr in ['zypper', 'community.general.zypper']
```
(See also #60)

**Comment by felixfontein** [comment]
@relrod thanks a lot for fixing this! :)

**Comment by relrod** [comment]
> Can you also re-enable the zypper_repository tests, i.e. revert #96 ?

Sorry, I sent this and went to get a few hours of sleep :). Thanks for adding that and merging, @felixfontein!

**Comment by felixfontein** [comment]
@relrod no problem, based on your location I assumed you finally went to bed and decided to simply update the PR ;)

**Pull request merged**